unity-renderer/Assets/Rendering/Culling/Tests/CullingControllerAndScenesShould.cs

```diff
@@ -80,5 +80,36 @@ yield return
 
             Assert.IsFalse(entity.meshesInfo.renderers[0].forceRenderingOff, "renderer wasn't brought back!");
         }
+
+        [UnityTest]
+        public IEnumerator StartStopCullingWithDisabledObjectsCorrectly()
+        {
+            var boxShape = TestHelpers.CreateEntityWithBoxShape(scene, Vector3.one * 1000, true);
+            var entity = boxShape.attachedEntities.First();
+
+            Assert.IsTrue(Environment.i.platform.cullingController.IsDirty(), "culling controller not dirty");
+            Assert.IsTrue(Environment.i.platform.cullingController.objectsTracker.IsDirty(), "object tracker not dirty");
+
+            yield return boxShape.routine;
+
+            yield return
+                new DCL.WaitUntil(() => entity.meshesInfo.renderers[0].forceRenderingOff, 0.3f);
+
+            Assert.IsTrue(entity.meshesInfo.renderers[0].forceRenderingOff, "renderer wasn't hidden!");
+
+            entity.gameObject.SetActive(false);
+            yield return null;
+
+            Environment.i.platform.cullingController.Stop();
+            TestHelpers.SetEntityTransform(scene, entity, Vector3.zero, Quaternion.identity, Vector3.one);
+            yield return null;
+
+            Environment.i.platform.cullingController.Start();
```

Comment:
```suggestion
```

We should immediately check if the entity was brought back. As the bug was that Stop() is not enabling the culling setting of disabled gameObjects.